tests/gocase/util/server.go

```diff
@@ -26,13 +26,16 @@ import (
 	"os"
 	"os/exec"
 	"path/filepath"
+	"syscall"
 	"testing"
 	"time"
 
 	"github.com/go-redis/redis/v9"
 	"github.com/stretchr/testify/require"
 )
 
+const k8sDefaultGracePeriod = 30
```

Comment:
Let's move this const to `util/constats.go` and bring the comment to this constant instead of inline.

Reply:
OK. Thanks for your help!